lib/jake.ex

```diff
@@ -11,46 +11,89 @@ defmodule Jake do
     "string"
   ]
 
-  def gen(%{"anyOf" => options} = spec) when is_list(options) do
+  def generator(map) do
+    StreamData.sized(fn size ->
+      Map.put(%{}, "map", map) |> Map.put("omap", map) |> Map.put("size", size) |> gen_init()
+    end)
+  end
+
+  def gen_init(schema) do
+    StreamData.bind(
+      get_lazy_streamkey(schema),
+      fn {nmap, nsize} ->
+        nschema = Map.put(schema, "map", nmap) |> Map.put("size", nsize)
+
+        if nmap["enum"] do
+          gen_enum(nschema, nmap["enum"])
```

Comment:
why is this being moved here? does it break on any special case?

Reply:
We can remove it from here.